### DISCUSSION TITLE
'bull-board' integration

### DISCUSSION THREAD

**Discussion by DaveLomber** [adonisjs/core#2084] [Help] [answered]
We are using 'bull' as for the queue solution with Adonis 4 https://github.com/OptimalBits/bull

There is a 'bull-board' express app to display the queue stats https://github.com/vcapretz/bull-board

This is a simple express app that can be injected into existing express app:

```
const app = require('express')()
const { router } = require('bull-board')

app.use('/admin/queues', router)
```

Since Adonis is based on top of express, can we inject it into Adonis app as well?

**Comment by RomainLanz** [comment, marked as answer]
Hey @DaveLomber! 👋 

I don't know where you learned that AdonisJS is based on ExpressJS, **because it is wrong**.
We are not based on any other framework.
